### PR TITLE
 Updating documentation to ease a bit when testing

### PR DESCRIPTION
**Pull request by chentex**
Signed-off-by: Vicente Zepeda Mas <vzepedam@redhat.com>

### Description

Updates the documentation to make it a bit easier to understand.
Adds examples using `CURL` to interact with the server

### CODE REVIEW

File: pod-compose.sh [hunk 1]
```diff
@@ -3,6 +3,7 @@
 set -o allexport
 source .env
 
+mkdir "$HOME/data_server/results" -p
```
Comment (Harshith-umesh) [inline pod-compose.sh]: can we check if it already exists ? and if not then go ahead and mkdir , else will throw an error

File: README.md [hunk 2]
```diff
@@ -120,15 +144,74 @@ Start snappy server pod with the image you built locally.
...
+
+#### Getting files
+
+You can get a file by providing the `filedir` and `filename` used in the upload
+
+```shell
+curl -X 'GET' -O 'http://localhost:7070/results/directory/name.tar'
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100 23964  100 23964    0     0  2340k      0 --:--:-- --:--:-- --:--:-- 2340k
```
Comment (Harshith-umesh) [inline README.md]: is this required in the readme?

Comment (chentex) [reply README.md]: Not necessary, was just part of the output example. if you want I can remove it.

**Comment by mfleader** [comment]
lgtm

**Pull request merged**